playout/libretime_playout/liquidsoap/client/_client.py

```diff
@@ -139,3 +142,11 @@ def settings_update(
                 self._set_var("message_offline", self._quote(message_offline))
             if input_fade_transition is not None:
                 self._set_var("input_fade_transition", input_fade_transition)
+            if default_crossfade_duration is not None:
+                self._set_var(
+                    "default_crossfade_transition", default_crossfade_duration
+                )
```

Comment:
This is failing with:

```
libretime-playout-1     | 2023-02-28 15:11:07,895 | ERROR    | libretime_playout.liquidsoap.client._client:_set_var:48 - Variable default_crossfade_transition is not defined.
```

Reply:
Thanks, I missed that. Initially I had the variables all named "default_crossfade_transition" but then changed it that is is consistent with the name in the DB and the Webpage

Reply:
Thanks, I missed that. 
Initially I had the variable named "default_crossfade_transition" but then changed it that is is consistent with the name in the DB and the Webpage.